src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -443,9 +458,9 @@ describe('containers/pages/NewPlan', () => {
     });
     document.body.appendChild(div);
     const wrapper = mount(
-      <Provider store={mockStore({})}>
+      <Provider store={mockStore(existingState)}>
         <Router history={history}>
-          <ConnectedBaseNewPlan />
+          <Route component={ConnectedBaseNewPlan} />
```

Comment:
Could you explain this change please?

Reply:
component expect to be used with route props and that is only possible if we pass the components itself to the route.

Reply:
component expect to be used with route props and that is only possible if we pass the components itself to the route